lib/faker/mock.ts

```diff
@@ -0,0 +1,246 @@
+import { faker } from "@faker-js/faker";
+import { Person } from "../types/person";
+import { z } from "zod";
+import {
+  OpportunitySchema,
+  PhaseSchema,
+  QuestionnaireSchema,
+} from "../schemas/opportunity";
+import { Question } from "../types/question";
+import db from "../../server/db";
+import { v4 as uuidv4 } from "uuid";
+import {
+  emailUniqueEnforcer,
+  keyUniqueEnforcer,
+  now,
+  program,
+  uuidUniqueEnforcer,
+} from "./utils";
+
+const options = program.opts();
+const numberOfUsers = parseInt(options.users, 10);
+const numberOfOpportunities = parseInt(options.opportunities, 10);
+const minNumberOfPhases = parseInt(options.minphases, 10);
+const maxNumberOfPhases = parseInt(options.maxphases, 10);
+const minNumberOfQuestionnaires = parseInt(options.minquestionnaires, 10);
+const maxNumberOfQuestionnaires = parseInt(options.maxquestionnaires, 10);
+
+console.log(minNumberOfPhases, maxNumberOfPhases)
+
+let generatedUsers: Person[];
+
+try {
+  await db.$connect;
+  await generateUsers(numberOfUsers);
+  const users = await fetchAllUsers();
+  console.log("All users:", users);
+  generatedUsers =
+    users?.map((user) => ({
+      id: user.id!,
+      name: user.name!,
+      image: user.image!,
+    })) || [];
+  console.log("Users generated!");
```

Comment:
Maybe for simplicity purposes, "Generated x users: [name1,name2,name3]" is more suitable than all user objects - reason being it may flood the command line with all the objects